### PR TITLE
Expose `close_to` up through `multifundchannel` + `fundchannel`

### PR DESCRIPTION
**Pull request by niftynei**
We've had `close_to` available on `fundchannel_start` for a while now. This wires it all the way through to general `fundchannel` users.

### CODE REVIEW

**Review by rustyrussell — APPROVED** [review]
Ack c256ce13dc01c242dec263742c1312d78168dac4

Some changes are in wrong commit though, needs rebase...

File: plugins/spender/multifundchannel.c [hunk 1]
```diff
@@ -516,6 +524,8 @@ param_destinations_array(struct command *cmd, const char *name,
 			   p_opt_def("announce", param_bool, &announce, true),
 			   p_opt_def("push_msat", param_msat, &push_msat,
 				     AMOUNT_MSAT(0)),
+			   /* FIXME: do address validation here */
```
[the remark this answers is not in the document]
Comment (niftynei) [reply plugins/spender/multifundchannel.c]: Left the fixme, added a note about validity and 'eventually failing'.

**Comment by niftynei** [comment]
Ok, misplaced changes have been moved to the correct commits; FIXME has been additionally detailed.

**Comment by rustyrussell** [comment]
Ack https://github.com/ElementsProject/lightning/pull/4132/commits/9650a869ffdd1b932ddfe0124308bd4e5e801267

**Pull request merged**